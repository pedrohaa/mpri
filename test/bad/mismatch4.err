File "bad/mismatch4.f", line 12, characters 6-21:
Type mismatch.
Expected: a
Inferred: a -> a
Hypotheses:
_ = a
__1_ = b
_ = __1_
