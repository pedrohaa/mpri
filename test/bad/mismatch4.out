forall a b. eq a b -> a
